### PR TITLE
DotPlot Chart & Some other charting components 

### PR DESCRIPTION
**Pull request by HHogg**
screen shot 2017-04-28 at 12 09 27

**Browser Checks**:
- [X] IE11
- [X] Edge
- [X] Chrome
- [X] Firefox
- [X] Safari

### CODE REVIEW

File: src/charts/chart-grid/ChartGrid.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@import '../../materials/colors';
+@import '../../materials/layout';
```
Comment (benji6) [inline src/charts/chart-grid/ChartGrid.css]: presumably this import isn't required?

File: src/charts/chart-grid/ChartGrid.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import './ChartGrid.css';
+
+export default class ChartGrid extends Component {
+  static propTypes = {
+    count: PropTypes.number.isRequired,
+  };
+
+  render() {
+    const { count, ...rest } = this.props;
+    const increments = Array.from({ length: count }, (_, i) => i * 10);
```
Comment (benji6) [inline src/charts/chart-grid/ChartGrid.js]: Looks familiar :D

File: src/charts/dot-plot/Dot.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { DataPoint } from 'bw-axiom';
+import omit from 'lodash.omit';
+
+export default class Dot extends Component {
+  static propTypes = {
+    percent: PropTypes.number.isRequired,
+  };
+
+  render() {
+    const { ...rest } = this.props;
```
Comment (benji6) [inline src/charts/dot-plot/Dot.js]: surely no point in shallow cloning `props` here?

File: src/charts/chart-grid/ChartGrid.css (hunk 1 above)
Comment (HHogg) [inline src/charts/chart-grid/ChartGrid.css]: This was my mistake, it was needed for making the grid lines responsive... just forgot to add that feature back in 😄. Done.

File: src/charts/chart-grid/ChartGrid.js (hunk 2 above)
Comment (jdelight) [inline src/charts/chart-grid/ChartGrid.js]: nice little pattern 👍

Comment (benji6) [inline src/charts/chart-grid/ChartGrid.js]: OK we can't actually use `Array.from` if we want to support IE11. You can do this instead to keep it ES5 compliant:

```js
Array.apply(null, { length: count }).map((_, i) => i * 10)
```

**Pull request merged**